src/sass_types/factory.cpp

```diff
@@ -45,26 +43,27 @@ namespace SassTypes
     }
   }
 
-  void Factory::initExports(Handle<Object> exports) {
-    Local<Object> types = NanNew<Object>();
-    exports->Set(NanNew("types"), types);
+  void Factory::initExports(v8::Local<v8::Object> exports) {
+    Nan::EscapableHandleScope scope;
```

Comment:
Scratch that, it is called from `RegisterModule` in `src/binding.cpp`. But given it is passed `node::ADDON_REGISTER_FUNCTION_ARGS_TYPE` parameter from Nan, I can (and probably should) use that macro.